app/src/main/java/com/sdp/movemeet/Activity/ListActivity.java

```diff
@@ -0,0 +1,102 @@
+package com.sdp.movemeet.Activity;
+
+import android.os.Bundle;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBarDrawerToggle;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.core.view.GravityCompat;
+import androidx.drawerlayout.widget.DrawerLayout;
+
+import com.google.android.material.navigation.NavigationView;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.sdp.movemeet.Backend.FirebaseInteraction;
+import com.sdp.movemeet.Navigation.Navigation;
+import com.sdp.movemeet.R;
+
+public class ListActivity extends AppCompatActivity {
```

Comment:
What is this activity used for ?

Reply:
I add a comment in my last commit, but I can explain more